src/components/Header/Header.js

```diff
@@ -0,0 +1,16 @@
+import styles from './Header.css'
+import Image from 'next/image'
+import Link from 'next/link'
+
+export default function Header() {
+    return (
+        <div className="header-container">
+            <Link href="/dashboard" style={{ textDecoration: "none", color: "blue"}}>
+                <div className="logo">XFLIX</div>
+            </Link>
+            <Link href="/login">
+                <button className="login-btn">로그인</button>
+            </Link>
```

Comment:
```suggestion
            <Link href="/login" className="login-btn">
                로그인
            </Link>
```

<Link/>가 이미 a 태그이기 때문에 안에 버튼을 굳이 넣을 필요는 없을 듯...!